utils/ErrorOutput.go

```diff
@@ -0,0 +1,46 @@
+package utils
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+)
+
+type ValidationError struct {
+	Field string
+	Error string
+}
+
+func ClientError(status int, body string) (events.APIGatewayProxyResponse, error) {
```

Comment:
this function does not check if the status parameter is within the range of 400 to 500. If the status parameter is outside of this range, then the function will return an events.APIGatewayProxyResponse with the wrong status code.

Reply:
same question, so resolving this @iamitprakash

Reply:
same question as below, so resolving this one.